src/common/globals.h

```diff
@@ -95,4 +95,6 @@ const int DEFAULT_POLYLIMIT = 16;
 
 const int DEFAULT_OSC_PORT_IN = 53280;
 const int DEFAULT_OSC_PORT_OUT = 53281;
+const float MIDI_MIN_FREQ = 8.1758;
```

Comment:
This is already tunings midi note 0 constant. And the other one is that times 1<<7 or what not so if possible I would prefer to not duplicate them